firmware/applications/SignBuddy/SignBuddy.proto

```diff
@@ -45,3 +53,11 @@ message Sample {
 
     required TouchData touchData = 4;
 }
+
+message Status {
```

Comment:
can we add a prefix to all of these things `Sample` `Status`  and `Message` aren't great types to have in our namespace.

maybe something like  `sbp_sensor_data`, `sbp_device_status` and `sbp_message` would be better where sbp stands for signbuddy packet or signbuddy proto

Reply:
Yeah sounds good, I'll address the naming in a separate PR because the changes will propagate a bit